src/features/leverage-tokens/leverageTokens.config.ts

```diff
@@ -474,6 +489,9 @@ export const leverageTokenConfigs: Record<string, LeverageTokenConfig> = {
       address: BASE_WETH,
       decimals: 18,
     },
+    lendingAdapter: {
+      address: '0x0000000000000000000000000000000000000000' as Address,
```

Comment:
need help populating this - not sure what vnet to check

Reply:
Can we just fetch the lending adapter dynamically?